### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-07-11

### DISCUSSION THREAD

**Discussion by xfiderek** [space-ros/space-ros#168] [General]
Previous agenda: https://github.com/space-ros/space-ros/discussions/166

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
- [ ] Review release follow-up issues and PRs
- [ ] Nominate work for next release
- [ ] Open next meeting discussion

**Comment by xfiderek** [comment]
1. Decide on naming convention for generate-repos.sh and friends - https://github.com/space-ros/space-ros/pull/164#issuecomment-2163635954
2. In the next release(s), can we consider making a `generate-repos` a util that can be installed via apt (same way as rosdep is installed)?

**Reply by xfiderek** [reply]
1. suggested naming convention from the linked comment has been approved - we need to raise another PR to change the name. @eholum, could you do that since you were working on this part? If not i can also take it :+1: 
2. Maybe in long term if the script will become more stable

**Comment by Bckempa** [comment]
Next agenda here: https://github.com/space-ros/space-ros/discussions/169
